services/QuillLMS/client/app/bundles/Session/startup/clientRegistration.js

```diff
@@ -2,5 +2,6 @@ import 'lazysizes';
 import 'lazysizes/plugins/parent-fit/ls.parent-fit';
 import ReactOnRails from 'react-on-rails';
 import NewAccountApp from './NewAccountAppClient.jsx';
+import FinishSetUpApp from './FinishSetUpAppClient';
 
-ReactOnRails.register({ NewAccountApp, });
+ReactOnRails.register({ NewAccountApp, FinishSetUpApp, });
```

Comment:
Oh, wow.  It never really occurred to me that you could pass multiple options into `ReactOnRails.register`.  Sweet!